### PR TITLE
Adding Americas caches to OSG

### PR DESCRIPTION
**Pull request by biozit**
Adding Americas caches to OSG

### CODE REVIEW

**Comment by DrDaveD** [comment]
Related to #181 .  For @jthiltges to approve, merge, and distribute.

File: etc/cvmfs/osgstorage-auth.conf [hunk 1]
```diff
@@ -21,6 +21,18 @@ CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://daejeon-kreonet-net.nationalrese
...
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://dtn-pas.denv.nrp.internet2.edu:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://dtn-pas.hous.nrp.internet2.edu:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://dtn-pas.jack.nrp.internet2.edu:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://mghpcc-cache.nationalresearchplatform.org:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://sdsc-cache.nationalresearchplatform.org:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://osdf-cache.sprace.org.br:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://amst-osdf-xcache01.es.net:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://lond-osdf-xcache01.es.net:8443/"
+
+
```
Comment (DrDaveD) [inline etc/cvmfs/osgstorage-auth.conf:35]: It would be better to not add extra blank lines (also in osgstorage.org.conf)

**Comment by biozit** [comment]
ok

**Review by jthiltges — CHANGES_REQUESTED** [review]

**Comment by biozit** [comment]
@jthiltges fixed! Good catch!

**Review by jthiltges — APPROVED** [review]
LGTM, thanks!

**Review by biozit — COMMENTED** [review]
done

**Comment by jthiltges** [comment]
Published to config-osg as rev 118.

**Pull request merged**